billing-service/src/main/resources/db/migration/main/V20210118093315__egbs_amendment_create_ddl.sql

```diff
@@ -0,0 +1,44 @@
+CREATE TABLE EGBS_AMENDMENT
+(
+   id CHARACTER VARYING (256) NOT NULL,
+   tenantid CHARACTER VARYING (256) NOT NULL,
+   amendmentId CHARACTER VARYING (256) NOT NULL,
+   businessservice CHARACTER VARYING (256) NOT NULL,
+   consumercode CHARACTER VARYING (256) NOT NULL,
+   amendmentReason CHARACTER VARYING (256) NOT NULL,
+   reasonDocumentNumber CHARACTER VARYING (256),
+   status CHARACTER VARYING (256) NOT NULL,
```

Comment:
Any other indexes needed?

Reply:
not used that much. the primary id is mostly used